src/tiler/styles.mss

```diff
@@ -44,126 +51,195 @@
    Ensure that any changes to which stream_order + zoom levels are rendered
    here have appropriate coinciding filters in server.js
 */
+
+#drb_streams_50 {
```

Comment:
As I mentioned in the comment, most of this file is modified to mirror the changes from #1499, so a lot of these diffs are irrelevant.